src/controllers/bundler.controller.ts

```diff
@@ -0,0 +1,100 @@
+import Ajv from 'ajv';
+import { NextFunction, Request, Response, Router } from 'express';
+import { ProblemException } from '../exceptions/problem.exception';
+import { Controller } from '../interfaces';
+import { BundlerService } from '../services/bundler.service';
+import { documentValidationMiddleware } from '../middlewares/document-validation.middleware';
+
+export class BundlerController implements Controller {
+  public basepath = '/bundle';
+  private ajv:Ajv;
+  bundlerService: BundlerService;
```

Comment:
You don't need the `ajv` instance. Please check my comment in another PR https://github.com/asyncapi/server-api/pull/69#discussion_r820571700. Also `bundlerService` is unnecessary because atm it contains only single method, so we can move that in this controller. I don't think so that we will use bundler in other paths.